src/main/java/org/jabref/gui/importer/actions/MergeReviewIntoComment.java

```diff
@@ -0,0 +1,23 @@
+package org.jabref.gui.importer.actions;
+
+import org.jabref.gui.BasePanel;
+import org.jabref.logic.importer.ParserResult;
+import org.jabref.logic.migrations.MergeReviewIntoCommentMigration;
+
+public class MergeReviewIntoComment implements GUIPostOpenAction {
```

Comment:
Maybe rename that with the action suffix so that it is better distingusishable from the other migration